include/aikido/perception/DetectedObject.hpp

```diff
@@ -0,0 +1,89 @@
+#ifndef AIKIDO_PERCEPTION_DETECTEDOBJECT_HPP_
+#define AIKIDO_PERCEPTION_DETECTEDOBJECT_HPP_
+
+#include <string>
+#include "aikido/io/yaml.hpp"
+
+namespace aikido {
+namespace perception {
+
+/// DetectedObject delegates a detected object from a third party
```

Comment:
Is "delegates" the right word here?
```suggestion
/// DetectedObject delegates a detected object from a third-party
```

Reply:
Here's a stab at new documentation:

DetectedObject stores information about objects detected by a third-party perception system. The perception system should communicate object information via a ROS visualization_msgs/Marker message.

The constructor parameters correspond to the following fields in the Marker message: ...

Reply:
Actually, maybe we should be mentioning the fields rather than constructor parameters here, since this is the class documentation.